doc/Makefile

```diff
@@ -69,6 +69,12 @@ clean:
 		$(MAKE) -C $$dir clean; \
 	done
 
+build-image:
+	docker build . -t ghcr.io/enterprisedb/barman/barman-pandoc:master
```

Comment:
maybe i'm wrong, but ghcr.io/enterprisedb images are not public, not sure this should be on an open-source project... as this will not work for someone that is not part of EDB

Reply:
I don't remember what is inside the internal image, maybe we can make the image public? I think so but not sure yet.